recipe/meta.yaml

```diff
@@ -95,18 +98,18 @@ outputs:
         - python-nvd3 >=0.15.0
         - python-slugify >=5.0
         - rfc3339-validator >=0.1.4
-        - rich >=12.4.4
         - rich-argparse >=1.0.0
+        - rich >=12.4.4
         - setproctitle >=1.1.8
         - sqlalchemy >=1.4.28,<2.0
         - sqlalchemy-jsonfield >=1.0
         - tabulate >=0.7.5
         - tenacity >=6.2.0,!=8.2.0
         - termcolor >=1.1.0
-        - typing-extensions >=4.0.0
         - unicodecsv >=0.14.1
         - universal_pathlib >=0.1.4
         - werkzeug >=2.0,<3
+        - apache-airflow-providers-common-io
```

Comment:
This is being added to conda-forge at:
https://github.com/conda-forge/staged-recipes/pull/25095

Reply:
This has been merged!